animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -112,6 +116,9 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
             self.newInput("Float", "End Size", "endSize", value = 0.5, minValue = 0)
             self.newInput("Float", "Start Angle", "startAngle", value = 0)
             self.newInput("Float", "End Angle", "endAngel", value = 6 * PI)
+            self.newInput("Float", "Spiral Height", "spiralHeight", value = 0)
+            self.newInput("Interpolation", "Spiral Interpolation", "spiralInterpolation", defaultDrawType = "PROPERTY_ONLY", hide = True)
```

Comment:
`Spiral Interpolation` is not very informative either. I suggest `Radius Interpolation`.